conda/MAESTRO/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: maestro
-  version: "1.2.1"
+  version: "1.2.2"
```

Comment:
Please check the requirements. Now some of the versions are too specific. I believe the 'most recent' versions should work. For example, the version for Python, numpy, scipy, macs2 and etc.

Reply:
@taoliu I tried to loosen the restriction for python, numpy and so on, but it failed when building the conda package. So I kept the original requirements.

Reply:
The reason that conda built on Travis failed is 'time out'. When new dependencies are required, Travis has to download the new packages instead of reusing the cache. It will take a significantly longer time.

Reply:
Travis successfully built conda package  of new dependencies: https://travis-ci.com/github/liulab-dfci/MAESTRO/builds/194816018 corresponding to ee1bee4

We would better include the new version of dependencies.